pyomo/contrib/doe/measurements.py

```diff
@@ -26,15 +26,15 @@
 #  ___________________________________________________________________________
 
 
-from pyomo.common.dependencies import (
-    numpy as np, numpy_available,
-    pandas as pd, pandas_available,
-    matplotlib as plt, matplotlib_available,
-)
+#from pyomo.common.dependencies import (
+#    numpy as np, numpy_available,
+#    pandas as pd, pandas_available,
+#    matplotlib as plt, matplotlib_available,
+#)
 
 
 class Measurements:
-    def __init__(self, measurement_index_time, variance=None, ind_string='_index_'):
+    def __init__(self, self_define_res=None, measurement_index_time=None, variance=None, ind_string='_index_'):
```

Comment:
After making the changes we discussed, update the doc string to include any new input arguments.